### ISSUE TITLE
Easier command-line interface

### ISSUE THREAD

**Issue by kevgo** [Originate/psapi#4]
__Proposed new command-line API__
```bash
psapi [--path="/books"] [--method="get"] [--response-status=200] [--response-data="..."] [config-file-name]
```

It can be used in two ways:
1. `psapi config-filename`
2. `psapi --path="/hello" --method="GET" --response-data="world"`

**Comment by alexpeachey** [comment]
It seems for all but the most trivial cases this could end up very unwieldy.
If we wanted to allow an on-the-fly construction of an API without the config file, perhaps these changes make more sense:

1. Remove the need for the `-r` flag and the cli just takes the name of the config file as an argument.
1. If a file name is not specified it reads from standard in.

This makes it in line with most unix tools and allows either crafting the config on the fly by running the cli and just typing in the config or if you have the config being generated from a source other than a static file, it could just be piped into the cli.

**Comment by alexpeachey** [comment]
So went to toy around with this a little and realized when reading from stdin, we couldn't know whether it is `json` or `cson`. So I'm thinking we would need a flag that says what format (optional if using a file with an appropriate extension).

So the cli could be invoked:

1. `psapi routes.cson` - Read the `routes.cson` file as cson.
1. `psapi --format=cson routes` - Read the `routes` file as cson.
1. `psapi --format=cson` - Read from stdin and expect cson.

**Comment by kevgo** [comment]
Cool, this is going into a good direction. For now I think we can go with (1) only. Then later we can add (3). I don't think (2) will make too much sense, at least for now.

**Comment by alexpeachey** [comment]
Addressed with PR #12
